### PR TITLE
testng-eclipse #225: update docs about dropping support for Eclipse 3.x

### PR DESCRIPTION
**Pull request by missedone**
update docs for https://github.com/cbeust/testng-eclipse/issues/225

### CODE REVIEW

**Comment by missedone** [comment]
ping @cbeust , @juherr

File: doc/download.html [hunk 1]
```diff
@@ -84,21 +84,24 @@ <h2 align="center">Downloading TestNG</h2>
...
-	</li>
-	<li>Make sure the check box next to URL is checked and click <i>Next</i>. 
-	</li>
-	<li>Eclipse will then guide you through the process. </li>
+  <li>Select<i> Help / Install new software.</i></li>
+  <li><i>Search for new features to install.</i></li>
+  <li><i>New remote site.</i></li>
+  <li>Eclipse 4.2 and above - Enter <a href="http://beust.com/eclipse">http://beust.com/eclipse</a>.</li>
+  <li>Eclipse 3.x - Not supported any more, please update your Eclipse to 4.2 or above.
+    <ul>
```
Comment (cbeust) [inline doc/download.html]: Just remove the whole "If you unfortunately" section, no point in giving recommendations on things we won't support.

File: doc/download.html [hunk 2]
```diff
@@ -84,21 +84,24 @@ <h2 align="center">Downloading TestNG</h2>
...
+      if you unfortunately have to stick with Eclipse 3.x, you may try below, but with no support:
+      <li>Eclipse 3.6, 3.7 - you still can try to install the latest version of plug-ins from update site <a href="http://beust.com/eclipse">http://beust.com/eclipse</a>; if it does not work, you can install older versions of the plug-ins from update site <a href="http://beust.com/eclipse-old">http://beust.com/eclipse-old</a>, at the time writing this guide, TestNG for Eclipse 6.9.10 still works.</li>
+      <li>Eclipse 3.4, 3.5 - unfortunately, no working plugin for them, please update to higer version of Eclipse and follow the corresponding guide.</li>
+      <li>Eclipse 3.3 and below - it's recommanded to update to higer version of Eclipse and follow the corresponding guide. <br/>
+      However, if it's not an option, you may try to install from <a href="http://beust.com/eclipse1">http://beust.com/eclipse1</a>.</li>
+    </ul>
+  </li>
+  <li>Make sure the check box next to URL is checked and click <i>Next</i>.</li>
```
Comment (cbeust) [inline doc/download.html]: End of removed section, keep "Make sure" and below.

Comment (missedone) [inline doc/download.html]: done

**Comment by juherr** [comment]
Maybe we should add:

```html
<a href="http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1549" class="drag" title="Drag to your running Eclipse workspace to install TestNG for Eclipse"><img class="img-responsive" src="http://marketplace.eclipse.org/sites/all/themes/solstice/_themes/solstice_marketplace/public/images/btn-install.png" alt="Drag to your running Eclipse workspace to install TestNG for Eclipse" /></a>
```

and/or a link to http://marketplace.eclipse.org/content/testng-eclipse

**Comment by missedone** [comment]
hi @juherr 
updated, how does it look now
thanks

**Comment by juherr** [comment]
:+1:

**Pull request merged**